README.md

```diff
@@ -1,16 +1,73 @@
-# verify-matching-service-test-tool
+# Verify Matching Service Test Tool
 
-An application to help test Verify local matching services
-([Source location](https://github.com/alphagov/verify-matching-service-test-tool.git))
+A [local matching service](http://alphagov.github.io/rp-onboarding-tech-docs/pages/ms/msWorks.html) allows you to find a match between a user’s verified identity and a record in your organisation's database(s).
 
-### How to use
+The Verify Matching Service Test Tool helps you check your local matching service can:
+* find and match records correctly
+* identify unmatched records correctly
 
-1. Download the zip [from here](https://github.com/alphagov/verify-matching-service-test-tool/releases)
-2. Unzip it
-3. Run `bin/verify-matching-service-test-tool`
+You can use the test tool while [building your local matching service](http://alphagov.github.io/rp-onboarding-tech-docs/pages/ms/msBuild.html) and include it any automated testing. 
 
-There are some tests executed as part of the distributed files. However,
-additional test cases can be added in `examples/match` and
-`examples/no-match` folders within the unzipped folder for match and
-no-match cases respectively. Those should be json files (examples can
-be found in `examples/match` and `examples/no-match` folders).
\ No newline at end of file
+## Prerequisites
+
+* Java 8
+
+## Installation
+
+1. Download the zip file from the [release list](https://github.com/alphagov/verify-matching-service-test-tool/releases)
+2. Unzip the file
+
+## Configuration
+
+Open the `verify-matching-service-test-tool.yml` file.
+
+Replace the `matchUrl` and `accountCreationUrl` URLs with the same URLs you created for your local matching service. Refer to the [guidance on matching requests for more information about matching and account creation URLs](http://alphagov.github.io/rp-onboarding-tech-docs/pages/ms/msBuild.html#respond-to-json-matching-requests).
+
+## Run
+
+Run `bin/verify-matching-service-test-tool` to start the test tool.
+
+This command will run a series of test scenarios. Successful results will return in green text. Failed results will return in red text with error messages.
+
+:question: Is this necessary to document?
```

Comment:
Probably not - could just say "will run a series of test scenarios" and segue in to the next section.